app/controllers/api/v0/cromerr_submit_transactions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V0::CromerrSubmitTransactionsController < ApiController
```

Comment:
We're getting a "submission to the cromerr signing service transaction" returned from the service. I'm not fond of any names for that concept, so "Submissions" is good enough if it makes more sense.